### PR TITLE
Andy's BU288 Notes for F23

### PR DESCRIPTION
**Pull request by Andoiiii**
Notes for content on midterm 1, midterm 2. No pdf for the final exam one exists, since i failed to procure one
Self written questions for midterm 1 content, with self-written answers

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -95,6 +95,12 @@ by Yash and Karthik.
 
 Another [flashcard set](https://quizlet.com/mian8680/folders/bu288-f2021/sets){:.a#BU288--Flashcards--Imaad} for the two midterms and final exam.
 
+#### BU 288 (F23) <small>Organizational Behaviour</small>
+
+[Notes for Midterm 1 content](./BU288/f23-mt1-notes.pdf), [Notes for Midterm 2 content](./BU288/f23-mt2-notes.pdf) by Andy. Unfortunately a set for the finals do not exist from him.
```
Comment (RetroCraft) [inline index.md:100]: can you add the tracking classes? it's just fun to have data :)

**Review by RetroCraft — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (Andoiiii) [reply index.md]: so thats what those do - should be added now

Comment (RetroCraft) [reply index.md]: needs to be double hyphens lol it's `course-code--category-name--author-name--title` where single hyphens get replaced by spaces. very stupid system that i cooked up at like 2am one night LOL

Comment (Andoiiii) [reply index.md]: i think i fixed it now?

**Comment by Andoiiii** [comment]
any update on this? should be good to go now, or do i need to make other fixes?

**Pull request merged**